libbeat/common/size.go

```diff
@@ -0,0 +1,18 @@
+package common
+
+import "github.com/dustin/go-humanize"
+
+// Size defines a new configuration option that will parse `go-humanize` compatible values into a
+// int64 when the suffix is valid or will fallback to bytes.
+type Size int64
+
+// Unpack convert a size defined from a human readable format into bytes.
+func (s *Size) Unpack(value string) error {
```

Comment:
I like this as a temporarely fix to make migration easier. But I would remove it again in 7.0 to only have 1 way to configure the values.

This would mean to when someone uses it without a unit, we should log a deprecation warning.

Reply:
I am OK to add a deprecation warning.

Reply:
@ruflin When I tried to use `cfgwarn` in that file, I hit a`import cycle not allowed`, to fix that I have either to move theses new common type out of `common`  OR maybe `cfgwarn` out of common and have a backward compatible alias? ...